core/metrics-core-impl/src/main/java/org/hawkular/metrics/core/impl/cassandra/MetricsServiceCassandra.java

```diff
@@ -800,6 +801,40 @@ private void dropKeyspace(String keyspace) {
         });
     }
 
+    @Override
```

Comment:
There might be an issue regarding the result when no data is found. In other places, null is returned instead of an empty list. Then in the JAX-RS module SimpleDataCallback checks for null and eventually returns a 204 response.